doc/source/user/howtos_index.rst

```diff
@@ -1,7 +1,7 @@
 .. _howtos:
 
 ################
-NumPy How Tos
+NumPy how-to's
```

Comment:
```suggestion
NumPy how-tos
```
Pretty sure this is how it is usually writen (and what we use mostly).

There are more of these in some files which would be nice to cover, but lets just put it in.

One note: Please avoid opening a PR from your `main` branch, it can lead to quite tedious situations (including loss of history).

Reply:
Hi @seberg, thank you for taking the time to review my PR! 

I understand your point about not opening a PR from the main branch and will be mindful of it in the future. I'm just starting out my journey as an open-source contributor and learning with experience. :)

As for the `how-tos` vs `how-to's`, I did some research and it looks like there is no definitive answer to this. The latter used to be the 'correct' way but the former is now increasingly being used. It seems to come down to a matter of stylistic choice. Some places like the [Macmillan Dictionary](https://www.macmillandictionary.com/dictionary/british/how-to-s) list it with an apostrophe. 

Anyway, my curiosity about grammar and linguistics aside, this is a trivial issue. I'm happy to incorporate your suggested change in my PR.

Reply:
Yeah, doesn't really matter, just looked like we mostly use the other form right now.